app/views/users/show.html.erb

```diff
@@ -0,0 +1,57 @@
+<h1>Show User Details</h1>
+<h3 class="user-id">User ID: <%= @user.id %></h3>
+
+<main id="user-details" class="page">
+  <div id="user-page">
+    <div class="btn-container">
+      <div class="home-link-container ">
+      <%= link_to "Home", root_path, class: "home-link2" %>
+      </div>
+      <div class="back-link-container ">
+        <%= link_to "Back", :back, class: "back-link2" %>
+      </div> 
+    </div>
+  </div>
+  
+
+  
+  <div class="user-details-container ">
+    
+    <div class=" user-item">
+       <div class="user-image">
+ <%= image_tag('user.png', width: "150px", height: "100px") %>
+ </div>
+ <div class="user-details">
+      <h2 class="user-name"><%= @user.name %></h2>
+      <p class="user-posts-counter">Number of posts: <%= @user.posts_counter %></p>
+      </div>
+    </div>
+
+    <article class="user-bio">
+      <div class="user-bio-content">
+      <h3>Bio</h3>
+        <%= @user.bio %>
+        </div>
+    </article>
+  </div>
+
+  <article class="user-posts">
+    <% @posts.each do |post| %>
```

Comment:
- [ ] Hi @Kel-nana, remember we were asked to reuse the methods previously created in our models. This method should render the user's 3 most recent posts. The posts should be render in descending order. The method in your model is call `#new_posts`. Please modify the method body from `asc` to `desc` :+1:

Reply:
implemented